examples/tutorials/mvdr_tutorial.py

```diff
@@ -38,258 +52,315 @@
-    reference_pow = reference.pow(2).mean(axis=1, keepdim=True)
-    mix_pow = (estimate * reference).mean(axis=1, keepdim=True)
-    scale = mix_pow / (reference_pow + epsilon)
-
-    reference = scale * reference
-    error = estimate - reference
-
-    reference_pow = reference.pow(2)
-    error_pow = error.pow(2)
+def get_irms(spec_clean, spec_noise):
+    mag_clean = spec_clean.abs() ** 2
+    mag_noise = spec_noise.abs() ** 2
+    irm_speech = mag_clean / (mag_clean + mag_noise)
+    irm_noise = mag_noise / (mag_clean + mag_noise)
+    return irm_speech[REFERENCE_CHANNEL], irm_noise[REFERENCE_CHANNEL]
 
-    reference_pow = reference_pow.mean(axis=1)
-    error_pow = error_pow.mean(axis=1)
 
-    sisdr = 10 * torch.log10(reference_pow) - 10 * torch.log10(error_pow)
-    return sisdr.item()
+irm_speech, irm_noise = get_irms(spectrum_clean, spectrum_noise)
 
 
 ######################################################################
-# Results
-# -------
+# 3.4.1. Visualize IRM of target speech
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
 
-######################################################################
-# Single-channel mask results
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-#
+plot_mask(irm_speech, "IRM of the Target Speech")
 
-for solution in results_single:
-    print(solution + ": ", si_sdr(results_single[solution][None, ...], reverb_clean[0:1]))
 
 ######################################################################
-# Multi-channel mask results
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# 3.4.2. Visualize IRM of noise
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
 
-for solution in results_multi:
-    print(solution + ": ", si_sdr(results_multi[solution][None, ...], reverb_clean[0:1]))
+plot_mask(irm_noise, "IRM of the Noise")
 
-######################################################################
-# Original audio
-# --------------
-#
 
 ######################################################################
-# Mixture speech
-# ~~~~~~~~~~~~~~
+# 4. Beamforming using SoudenMVDR
+# -------------------------------
 #
 
-ipd.Audio(mix[0], rate=16000)
 
 ######################################################################
-# Noise
-# ~~~~~
+# 4.1. Compute PSD matrices
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #
+# :py:func:`torchaudio.transforms.PSD` computes the time-invariant PSD matrix given
+# the multi-channel complex-valued spectrum and the Time-Frequency mask.
+#
+# The shape of the PSD matrix is `(..., freq, channel, channel)`.
+
+psd_transform = torchaudio.transforms.PSD()
+
+psd_speech = psd_transform(spectrum_mix, irm_speech)
+psd_noise = psd_transform(spectrum_mix, irm_noise)
 
-ipd.Audio(noise[0], rate=16000)
 
 ######################################################################
-# Clean speech
-# ~~~~~~~~~~~~
+# 4.2. Apply Beamforming
+# ~~~~~~~~~~~~~~~~~~~~~~
+#
+# :py:func:`torchaudio.transforms.SoudenMVDR` takes the multi-channel
+# complexed-valued spectrum of the mixture speech, PSD matrices of
```

Comment:
in general, throughout the tutorial, we should be more precise with distinguishing between "spectrum" and "spectrogram"

```suggestion
# complex-valued spectrogram of the mixture speech, PSD matrices of
```

Reply:
all of the feedback in this section also applies to "5.3. Apply Beamforming"